src/components/shared/PlanSelector.tsx

```diff
@@ -41,7 +41,7 @@ const PlanMenu: FunctionalComponent<PlanMenuProps> = ({
               setAllConfiguredFeatures(configurableFeatureDefaults(plans as PlanEdge[], plan.id));
             }}
           />
-          <PlanCard plan={plan} isChecked={plan.id === selectedPlanId} />
+          <PlanCard plan={plan} isChecked={plan.id === selectedPlanId} calculatedCost={0} />
```

Comment:
Why is this zero?

Reply:
it might not be the most obvious thing, but the way it behaves is if calculated is `0` it will fall back to the plan base cost. Maybe that should be `null` or something. more. explicit

Reply:
fixed this:
- number: it has a calculated cost
- undefined: fall back to plan base cost
- null: loading